vms/platformvm/state/diff.go

```diff
@@ -562,5 +566,8 @@ func (d *diff) Apply(baseState State) error {
 			baseState.DeleteUTXO(utxoID)
 		}
 	}
+	for subnetID, owner := range d.subnetOwners {
+		baseState.SetSubnetOwner(subnetID, owner)
+	}
```

Comment:
This is a really weird asymmetry. If I call `AddSubnet` on a `Diff` that will call `AddSubnet` and `SetSubnetOwner` on the base state.

Reply:
Adjusted it to be explicitly set when calling `AddSubnet`